src/js/cljs.js

```diff
@@ -460,6 +460,17 @@ async function startClojureScriptEngine(opts: CLIOptsType): Promise<mixed> {
 
     startREPL(opts);
   }
+
+  if (!mainNsName && !repl) {
+    initClojureScriptEngine(opts);
+    execute(
+      "(require 'lumo.repl) (lumo.repl/run-main-cli-fn)",
```

Comment:
we should probably just create a JS function that calls `lumo.repl/run-main-cli-fn` like we do above for `runMain`